cli/job.go

```diff
@@ -88,11 +88,9 @@ func runJobCmd() *cobra.Command {
 			)
 
 			jobsMap := map[string]func(context.Context) error{
-				"fetch_resources":              handler.FetchResources,
-				"grant_expiration_reminder":    handler.GrantExpirationReminder,
-				"grant_expiration_revocation":  handler.RevokeExpiredGrants,
-				"appeal_expiration_reminder":   handler.GrantExpirationReminder,
-				"appeal_expiration_revocation": handler.RevokeExpiredGrants,
```

Comment:
This would also mean chart changes in here - https://github.com/odpf/charts/blob/main/stable/guardian/values.yaml 🤔